src/test/ui/regions/regions-close-object-into-object-2.nll.stderr

```diff
@@ -1,13 +1,13 @@
 error: lifetime may not live long enough
```

Comment:
We lose several suggestions here.

Reply:
Created https://github.com/rust-lang/rust/issues/96277 for suggestions around trait object lifetime bound